library/system.lisp

```diff
@@ -62,6 +68,64 @@ While the result will always contain microseconds, some implementations may retu
       (cl:sleep n)
       Unit)))
 
+;;;
+;;; Pofiling
+;;;
+
+(coalton-toplevel
+
+  (declare get-run-time (Unit -> UFix))
+  (define (get-run-time)
+    "Gets the run-time."
+    (lisp UFix ()
+      (cl:get-internal-run-time)))
+
+  (declare get-real-time (Unit -> UFix))
+  (define (get-real-time)
+    "Gets the real-time."
+    (lisp UFix ()
+      (cl:get-internal-real-time)))
+
+  #+sbcl
```

Comment:
please do not do this in the standard library

Reply:
won't happen again